.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Report a bug
+title: "[BUG] - "
+labels: bug
+assignees: ''
+
+---
+
+**Internal/External**
+*Internal*   if an IOHK staff member.
+*External*   otherwise.
+
+**Area**
+*PlutusTx*    Related to the GHC plugin, Haskell-to-Plutus compiler, on-code
+*PAB*    Related to the Plutus application backend (PAB), emulator, Plutus libraries
+*Marlowe* Related to Marlowe
+*Other*   Any other topic (Playgrounds, etc.)
```

Comment:
It's a shame you can't do conditional tagging based on checkboxes or something...

Reply:
Also I wonder if we should use one of our more "product"-y breakdowns, e.g. "PlutusTx -> Plutus Foundation", "PAB -> Plutus Application Framework"?

Reply:
Yeah good idea.